.vale.ini

```diff
@@ -0,0 +1,5 @@
+StylesPath = .vale/styles
+MinAlertLevel = suggestion
+
+[*.md]
+BasedOnStyles = Microsoft, write-good
```

Comment:
Please add a new line.

Reply:
fixed with https://github.com/wso2/docs-is/pull/5330/commits/18d7e412e7b9f7a7073c02b6bc631a913b6c8e73